lib/workers/repository/update/pr/index.ts

```diff
@@ -316,7 +316,10 @@ export async function ensurePr(
             });
           }
         }
-      } else if (logJSON.error === ChangeLogError.MissingGithubToken) {
+      } else if (
+        logJSON.error === ChangeLogError.MissingGithubToken &&
```

Comment:
Potentially this error should be suppressed at the source instead. E.g. it returns null or empty release notes instead of an error